source/Sync/EMethodPatchFlag.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Sync
+{
+    [Flags]
+    public enum EMethodPatchFlag : byte
+    {
+        None = 0,
+        TransferArgumentsByValue = 1 << 1
```

Comment:
It was a little convoluted on what that bit shifting meant. Figured it out thought. Can you add "Reserved" for the other 6 bits?